Competitive Coding/Strings/String Search/SuffixArray/Suffix_Array.md

```diff
@@ -0,0 +1,27 @@
+This is the code to build suffix array for any given string input.
+
+Suffix Array:-
+A suffix array for a given string is basically a sorted array of all suffixes of a string.
+
+For example:-
+
+Given the string is "banana"
+
+The suffixes are:-
+
+0 banana			5 a
```

Comment:
make this part more look good.

Like codify this part to make it look good. Otherwise this part looks odd